### PR TITLE
📖Add port security doc

### PR DESCRIPTION
**Pull request by jichenjc**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**TODOs**:

- [ ] squashed commits
- if necessary:
  - [ ] includes documentation
  - [ ] adds unit tests

/hold

### CODE REVIEW

**Comment by hidekazuna** [comment]
PTAL @iamemilio

File: docs/book/src/clusteropenstack/configuration.md [hunk 1]
```diff
@@ -231,6 +231,22 @@ spec:
...
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
+kind: OpenStackMachine
+metadata:
+  name: <cluster-name>-controlplane
+  namespace: <cluster-name>
+spec:
+  ports:
+  - networkId: <your-network-id>
+    ...
+    disablePortSecurity: True
```
Comment (tobiasgiese) [inline docs/book/src/clusteropenstack/configuration.md:246]:
```suggestion
    disablePortSecurity: true
```

We always write booleans in lower case in yaml. I would recommend keeping it consistent.

**Comment by iamemilio** [comment]
Thanks! Looks good to me.

**Pull request merged**